calisphere/record.py

```diff
@@ -0,0 +1,217 @@
+from copy import deepcopy
+from .es_cache_retry import json_loads_url
+from typing import Any, Dict
+from django.conf import settings
+from django.http import Http404
+from .collection_views import Collection
+from .institution_views import Repository
+
+def get_iiif(media_id) -> Dict[str, Any]:
+    iiif_url = f"{settings.UCLDC_IIIF}{media_id}/info.json"
```

Comment:
```suggestion
    iiif_url = f"{settings.UCLDC_IIIF}{urllib.parse.quote(media_key)}/info.json"
```

Reply:
I made this change here as part of #295 here: https://github.com/ucldc/public_interface/pull/295/commits/86c31c69cd177b0589db9ed22eed85d2b6f9d696